lambda-resume-builder/src/Components/Dashboard.js

```diff
@@ -15,6 +15,11 @@ import MenuIcon from '@material-ui/icons/Menu';
 import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
 import NotificationsIcon from '@material-ui/icons/Notifications';
 import { mainListItems, secondaryListItems } from './listItems';
+import {BrowserRouter as Router} from 'react-router-dom';
+import {Route} from 'react-router-dom';
+import Calendar from '../Components/Calendar/Calendar';
+import Assignments from '../Components/Assignments/Assignments';
+import Endorsement from '../Components/EndorsementChecklist/EndorsementChecklist';
```

Comment:
Why does `Endorsement` _not_ follow the same naming convention as the file name?